app/services/update_duplicate_matches.rb

```diff
@@ -21,19 +21,20 @@ def save!
 private
 
   def save_match(match)
-    fraud_match, candidate = nil
     ActiveRecord::Base.transaction do
-      fraud_match, candidate = create_or_update_fraud_match(match)
+      candidate = create_or_update_fraud_match(match)
       candidate.update!(submission_blocked: true)
     end
-    notify_candidate(candidate, fraud_match)
   end
 
   def create_or_update_fraud_match(match)
     fraud_match = FraudMatch.find_by(last_name: match['last_name'], postcode: match['postcode'], date_of_birth: match['date_of_birth'])
     candidate = Candidate.find(match['candidate_id'])
```

Comment:
side note: the naming of the variables here is a bit confusing: `fraud_match` which is a `FraudMatch` and `match` which is the matched data

Reply:
Yes, exactly! It is confusing. Do you have a suggestion?

Reply:
I guess it's a new_match and the one above is an existing_match?

Reply:
But we won't know that is an existing fraud match until we do the if below `if fraud_match.present?`.
Maybe I can rename the code inside the else to new_match only?

Reply:
Done! ♻️